boards/beagle/beagleplay_cc1352/doc/index.rst

```diff
@@ -0,0 +1,193 @@
++-------+--------------+-------------------------------------+
+| DIO13 | CC1352_TX    | AM62 UART6_RXD                      |
++-------+--------------+-------------------------------------+
+| DIO14 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO15 | CC1352_BOOT  | AM62 GPIO0_13                       |
++-------+--------------+-------------------------------------+
+| DIO16 | CC1352_TDO   | TAG-CONNECT TDO                     |
++-------+--------------+-------------------------------------+
+| DIO17 | CC1352_TDI   | TAG-CONNECT TDI                     |
++-------+--------------+-------------------------------------+
+| DIO18 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO19 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO20 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO21 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO22 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO23 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO24 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO25 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO26 | N/C          |                                     |
++-------+--------------+-------------------------------------+
+| DIO27 | LED1         | CC1352_LED1 yellow LED9             |
++-------+--------------+-------------------------------------+
+| DIO28 | LED2         | CC1352_LED2 yellow LED8             |
++-------+--------------+-------------------------------------+
+| DIO29 | RF_PA        | Antenna mux PA enable               |
++-------+--------------+-------------------------------------+
+| DIO30 | RF_SUB1G     | Antenna mux SubG enable             |
++-------+--------------+-------------------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To flash, disable the existing driver that ties up the serial port and use
+the customized BSL Python script.
+
+1. Ensure the bcfserial driver isn’t blocking the serial port.
+
+   .. code-block:: console
+
+      echo "    fdtoverlays /overlays/k3-am625-beagleplay-bcfserial-no-firmware.dtbo" | sudo tee -a /boot/firmware/extlinux/extlinux.conf
+
+2. Now reboot the board.
+
+   .. code-block:: console
+
+      sudo shutdown -r now
+
+3. Install CC1352-flasher
+
+   .. code-block:: console
+
+      pip install cc1352-flasher
```

Comment:
Note: on recent images, this should already be installed. It would be good to make sure it is.